vault/datadog_checks/vault/vault.py

```diff
@@ -122,29 +124,34 @@ def get_config(self, instance):
             password = instance.get('password')
             config['auth'] = (username, password) if username and password else None
 
-            config['ssl_verify'] = _is_affirmative(instance.get('ssl_verify', True))
+            config['ssl_verify'] = is_affirmative(instance.get('ssl_verify', True))
+            config['ssl_ignore_warning'] = is_affirmative(instance.get('ssl_ignore_warning', False))
             config['proxies'] = self.get_instance_proxy(instance, config['api_url'])
             config['timeout'] = int(instance.get('timeout', 20))
             config['tags'] = instance.get('tags', [])
 
             # Keep track of the previous cluster leader to detect changes.
             config['leader'] = None
-            config['detect_leader'] = _is_affirmative(instance.get('detect_leader'))
+            config['detect_leader'] = is_affirmative(instance.get('detect_leader'))
 
             self.config[instance_id] = config
 
         return config
 
     def access_api(self, url, config, tags):
         try:
-            response = requests.get(
-                url,
-                auth=config['auth'],
-                verify=config['ssl_verify'],
-                proxies=config['proxies'],
-                timeout=config['timeout'],
-                headers=config['headers']
-            )
+            with warnings.catch_warnings():
+                if config['ssl_ignore_warning']:
```

Comment:
We have this as `ignore_ssl_warning` in places like the Http check. Can we rename this for consistency?

Reply:
I think we're moving toward a `category_*` approach.